projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -656,6 +657,17 @@ public function handle_preferred_view() {
 		// phpcs:enable WordPress.Security.NonceVerification
 	}
 
+	/**
+	 * Adds the necessary CSS class to the admin body class.
+	 *
+	 * @param string $admin_body_classes Contains all the admin body classes.
+	 *
+	 * @return string
+	 */
+	public function admin_body_class( $admin_body_classes ) {
+		return " nav-unification-enabled $admin_body_classes ";
```

Comment:
Nitpick: What do you think about using `is-nav-unification` instead, so it matches [the class used in Calypso](https://github.com/Automattic/wp-calypso/blob/c67e6557881c66ca0658c85e75c84db0b2e7f28d/client/layout/index.jsx#L237)?
```suggestion
		return " is-nav-unification $admin_body_classes ";
```

Reply:
I'll apply the change thanks!